docker/Dockerfile

```diff
@@ -161,45 +162,9 @@ ENV NETWORK ${NETWORK:-Mainnet}
 ARG RPC_PORT
 ENV RPC_PORT ${RPC_PORT}
 
-# Set this to log to a file, if not set, logs to standard output
-ARG LOG_FILE
-ENV LOG_FILE ${LOG_FILE}
-
-# Set this to change the default cached state directory
-ARG ZEBRA_CACHED_STATE_DIR
-ENV ZEBRA_CACHED_STATE_DIR ${ZEBRA_CACHED_STATE_DIR:-/var/cache/zebrad-cache}
-
-# Zebra automatically detects if it is attached to a terminal, and uses colored output.
-# Set this to 'true' to force using color even if the output is not a terminal.
-# Set this to 'false' to disable using color even if the output is a terminal.
-ARG LOG_COLOR
-ENV LOG_COLOR ${LOG_COLOR}
-
 # Expose configured ports
-
 EXPOSE 8233 18233 $RPC_PORT
```

Comment:
@teor2345 suggests we let users set their own ports in `runtime-entrypoint.sh`, which is inconsistent with this instruction. Plus, some ports are missing, and at least one will always be redundant.

I guess we could just get rid of it completely, since

> The `EXPOSE` instruction does not actually publish the port. It functions as a type of documentation between the person who builds the image and the person who runs the container, about which ports are intended to be published.

The `EXPOSE` instruction is useful for automated builds using the `-P` flag, but I don't think it makes much sense for Zebra, since

> The `-P` option publishes all the ports to the host interfaces. Docker binds each exposed port to a random port on the host.

Reply:
Also, the exposed `$RPC_PORT` can differ from the one used in `runtime-entrypoint.sh`.

Reply:
Thanks, I wasn't expecting it to work that way!

But I was going to ask a question about updating EXPOSE.

Reply:
Clarification: I took the cited parts from the official docs.

Reply:
I'm just leaving `8233 18233` as

> It functions as a type of documentation between the person who builds the image and the person who runs the container, about which ports are intended to be published.